src/control/server/mgmt_check.go

```diff
@@ -245,9 +248,17 @@ func (svc *mgmtSvc) SystemCheckStart(ctx context.Context, req *mgmtpb.CheckStart
 	}
 
 	if resp.Status > 0 {
-		svc.log.Debug("resetting checker findings DB")
-		if err := svc.sysdb.ResetCheckerData(); err != nil {
-			return nil, errors.Wrap(err, "failed to reset checker finding database")
+		if len(req.Uuids) == 0 {
```

Comment:
should the command also accept labels?

Reply:
The incoming request (via grpc) can include labels. They are translated to UUIDs in `makePoolCheckerCall`. I don't really like the way the Uuids field is overloaded for both of these, but that's a bit outside the scope of this PR.